ui/app/routes/vault/cluster/secrets/backend/configuration.js

```diff
@@ -23,7 +22,6 @@ export default Route.extend({
           'deleteVersionAfter',
           backend.deleteVersionAfter !== '0s' ? backend.deleteVersionAfter : 'Never delete'
         );
-        backend.set('maxVersions', backend.maxVersions ? backend.maxVersions : 'Not set');
```

Comment:
Curious why we no longer need this? Is the _Not set_ placeholder handled somewhere else?

Reply:
Good catch. This was something that was handled in a separate issue with KV. I'll spare you some complicated details about the parent engine overriding when this is set to 0. Basically, I removed so this now matches the KV config view.